src/python/setup.py

```diff
@@ -132,6 +132,7 @@ def run(self):
         "pillow >= 5.2.0",
         "prettytable == 0.7.2",
         "resampy == 0.2.1",
+        "llvmlite == 0.31.0",
```

Comment:
Do you think it would be better to only pin this for Python 2.7?

Reply:
Good call.  Fixed.